packages/subgraph/scripts/deploy-subgraph.sh

```diff
@@ -45,8 +45,8 @@ then
         --node http://localhost:8020
 else
     graph deploy $FULLNAME \
-        --version-label $SUBGRAPH_VERSION \
-        --node https://app.satsuma.xyz/api/subgraphs/deploy \
+        --ipfs https://api.thegraph.com/ipfs/ \
+        --node https://api.thegraph.com/deploy/ \
```

Comment:
revert that back to satsuma before merge

Reply:
done